org.eclipse.sisu.plexus/pom.xml

```diff
@@ -64,6 +69,20 @@
       <artifactId>org.eclipse.sisu.inject</artifactId>
       <version>${project.version}</version>
     </dependency>
```

Comment:
a mandatory dependency to javax.inject is necessary as well, as e.g. `javax.inject.Provider` is used in `PlexusRequirements`